example/src/bin/docs_examples.rs

```diff
@@ -468,28 +372,50 @@ pub async fn example_API_ListRemoveValue(
     cache_client: &CacheClient,
     cache_name: &String,
 ) -> Result<(), MomentoError> {
-    match cache_client
+    cache_client
         .list_remove_value(cache_name, "list_name", "value1")
-        .await
-    {
-        Ok(_) => println!("Successfully removed value"),
-        Err(e) => eprintln!("Error removing value: {:?}", e),
-    }
+        .await?;
+    println!("Value removed from list");
     Ok(())
 }
 
 #[allow(non_snake_case)]
-pub async fn example_API_DictionarySetFields(
+pub async fn example_API_DictionaryIncrement(
     cache_client: &CacheClient,
     cache_name: &String,
 ) -> Result<(), MomentoError> {
-    match cache_client
+    cache_client
+        .dictionary_increment(cache_name, "dictionary_name", "field", 1)
+        .await?;
+    println!("Incremented field in dictionary");
```

Comment:
Same comment here re: incremented value in response.

Reply:
oh good catch, will update